.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,45 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/dotnet
+{
+	"name": "C# (.NET)",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/dotnet:1-8.0-bookworm",
+	"features": {
+		"ghcr.io/devcontainers/features/docker-in-docker:2": {},
+		"ghcr.io/devcontainers/features/powershell:1": {},
+		"ghcr.io/dapr/cli/dapr-cli": {
+			"version": "1.12.0"
+		}
+	},
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"ms-dotnettools.vscodeintellicode-csharp",
+				"chneau.resource-monitor"
```

Comment:
I'm not familiar with this. Is it the most mainstream option? Doesn't seem super popular.

https://marketplace.visualstudio.com/items?itemName=chneau.resource-monitor

Reply:
oh right sorry. It's a personal preference to see if the dev container hangs because it hangs or if it really does something. I will remove it